src/_tkagg.cpp

```diff
@@ -326,7 +326,6 @@ int load_tkinter_funcs(void)
  * tkinter uses these symbols, and the symbols are therefore visible in the
  * tkinter dynamic library (module).
  */
-#if PY_MAJOR_VERSION >= 3
```

Comment:
just inline the conditional stuff below?

Reply:
OK, will do when I next rebase.